ModTek/ModTek.cs

```diff
@@ -745,6 +774,19 @@ IEnumerator<ProgressReport> NestedFunc()
             ProgressPanel.SubmitWork(NestedFunc);
         }
 
+        internal static string ResolvePath(string path)
+        {
+            path = path.Replace("$Mods", ModsDirectory);
+            //path = path.Replace("$StreamingAssets", StreamingAssetsDirectory);
```

Comment:
delete?

Reply:
sure, I was struggling if it should be explicit or implicit, but its unlikely that the explicit syntax will ever be used in the first place, so no need to have it redundantly as both